### ISSUE TITLE
Update didn't fix gogoanime server issue.

### ISSUE THREAD

**Issue by ChaosGodEmperor** [kl3jvi/animity#143]
![Screenshot_20231130_222827_Animity](https://github.com/kl3jvi/animity/assets/98984322/1f799ad7-44de-4cbc-b780-64f684bd84e1)

**Comment by kl3jvi** [comment]
Hi, it's working on my end though

**Comment by professorDeveloper** [comment]
Hi bro

**Comment by ChaosGodEmperor** [comment]
yeah, still same issue. Uninstalled and reinstalled it multiple times. It's only when I click on an episode to play that it show this. I am using the latest 1.2.2 version of the app.

**Comment by obi-o-n-e** [comment]
Same issue,
Can confirm
